src/javascript/app/base/binary_pages.js

```diff
@@ -65,86 +66,89 @@ const WhyUs              = require('../../static/pages/why_us');
 
 /* eslint-disable max-len */
 const pages_config = {
-    account_transfer         : { module: AccountTransfer,            is_authenticated: true, only_real: true, needs_currency: true },
     accounts                 : { module: Accounts,                   is_authenticated: true, needs_currency: true },
     api_tokenws              : { module: APIToken,                   is_authenticated: true },
+    authenticate             : { module: Authenticate,               is_authenticated: true, only_real: true },
```

Comment:
`u` is after `s`, so this should remain on line 73. we sort alphabetically by keys, and `authenticate` comes after `asset_indexws` but before `authorised_appsws`. the placement of `account_transfer` is wrong, it should be on line 68 as before. since `account_` comes before `accounts`